decidim-admin/app/packs/src/decidim/admin/choose_language.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable no-invalid-this */
+/* eslint-disable require-jsdoc */
 
-$(() => {
-  $("select.language-change").change(function () {
-    let $select = $(this);
-    let targetTabPaneSelector = $select.val();
-    let $tabsContent = $select.parent().parent().siblings();
+export default function initLanguageChangeSelect(elements) {
+  elements.forEach((select) => {
+    select.onchange = function () {
+      let targetTabPaneSelector = this.value;
+      let $tabsContent = this.parentElement.parentElement.nextElementSibling;
```

Comment:
Here also `select` instead of `this`. Also, let's remove the dollar sign from the element name as it's no longer a jQuery element (the dollar sign is generally used to refer to jQuery elements).
```suggestion
      let tabsContent = select.parentElement.parentElement.nextElementSibling;
```

Reply:
:+1: